renovate.json

```diff
@@ -1,6 +1,5 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "local>descope/renovate-config"
-  ]
+  "extends": ["local>descope/renovate-config"],
+  "enabled": false
```

Comment:
renovate disabled completely here ☝️ need to revisit later

Reply:
@omercnet we will still get security issues from Dependabot, right?